internal/storage/caching/fast_stat_bucket.go

```diff
@@ -97,6 +97,15 @@ func (b *fastStatBucket) addNegativeEntry(name string) {
 	b.cache.AddNegativeEntry(name, expiration)
 }
 
+// LOCKS_EXCLUDED(b.mu)
+func (b *fastStatBucket) addNegativeEntryForFolder(name string) {
```

Comment:
should we write unit test for this?

Reply:
I don't think so, it is just calling cache `AddNegativeEntryForFolder` only with expiration which have unit test in stat cache file. There are similar method in the file also.

Reply:
Okay, atleast we are testing as part of public bucket method.